Tools/setup.py

```diff
@@ -130,10 +122,29 @@ def run(self):
         sys.exit()
 
 
+# Versioning magic.
+VERSION_FILE_PATH = Path('dea_tools/__version__.py')
+about = {}
+# Try to read the version from git.
+try:
+    version = setuptools_scm.get_version(
+        root="..",
+        write_to='Tools' / VERSION_FILE_PATH,
+        relative_to=__file__)
+except LookupError:
+    try:
+        # no .git folder, so read from __version__.py
+        with open(VERSION_FILE_PATH) as f_version:
+            exec(f_version.read(), about)
+            version = about['version']
+    except FileNotFoundError:
+        # No __version__.py, yikes
+        raise RuntimeError('Build in-place with --use-feature=in-tree-build.')
+
```

Comment:
You can just configure `setuptools_scm` in `pyproject.toml` like here

https://github.com/opendatacube/datacube-core/blob/dd46ceb7e027b1900547cf70696c72e4aa8ce552/pyproject.toml#L5-L6

```toml
[tool.setuptools_scm]
write_to="__version__.py"
root=".."
```

Reply:
Only if --use-feature=in-build-tree is enabled; this doesn't exist on Sandbox as pip is outdated. Otherwise pip copies the whole directory to /tmp and loses the git info

Reply:
Unless you just mean the configuration of setuptools_scm, in which case yeah that could be in the pyproject.toml